pages/know-your-rights/index.tsx

```diff
@@ -0,0 +1,177 @@
+import Head from 'next/head'
```

Comment:
General thoughts: can we center this page?
This is still pending addition of Spanish language copy, yes?